mojio_sdk/api.py

```diff
@@ -48,6 +49,28 @@ def get_vehicles(self):
         data = response.json()
         return self._parse_devices(data)
 
+    def get_trips(self):
+        if self._authentication is None or not self._authentication.is_valid():
+            self.login()
```

Comment:
If you wanted to improve the code, you'd refactor the duplicate into a function `login_if_no_session`